packages/grafana-ui/src/components/QueryField/QueryField.tsx

```diff
@@ -18,6 +18,10 @@ import {
 
 import { makeValue, SCHEMA, CompletionItemGroup, TypeaheadOutput, TypeaheadInput, SuggestionsState } from '../..';
 import { selectors } from '@grafana/e2e-selectors';
+import { css, cx } from '@emotion/css';
+import { GrafanaTheme2 } from '@grafana/data';
+import { withTheme2 } from '../../themes';
+import { getFocusStyles } from '../../themes/mixins';
 
 export interface QueryFieldProps {
```

Comment:
```suggestion
export interface QueryFieldProps  extends Themeable2 { 
```
Nit: 
🤔 I think when we use `withTheme2` we should use the `themeable2` interface,  you can take a look at the style guide [recommendation](https://github.com/grafana/grafana/blob/main/contribute/style-guides/themes.md#using-themecontext-directly)

Reply:
Nice catch, this is my first time adding styles to a class component in Grafana 😅